examples/crds/ciliumnodes.yaml

```diff
@@ -253,6 +253,11 @@ spec:
                     items:
                       description: AzureInterface represents an Azure Interface
                       properties:
+                        GatewayIP:
```

Comment:
Any particular reason why this field is capitalized?

Reply:
@ti-mo please refer to the above discussion with @aanm. This property-name is generated from JSON tag in `AzureInterface` struct in pkg/azure/types/types.go. We need to keep this capitalized so that it doesn't break the API.